plugins/base/src/main/resources/edu/wpi/first/shuffleboard/plugin/base/widget/RelayWidget.fxml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ToggleButton?>
+<?import javafx.scene.control.ToggleGroup?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<StackPane fx:id="root" stylesheets="@relay-widget.css" xmlns="http://javafx.com/javafx/8.0.121"
+           xmlns:fx="http://javafx.com/fxml/1"
+           fx:controller="edu.wpi.first.shuffleboard.plugin.base.widget.RelayWidget">
```

Comment:
No minimum size set, makes the widget look odd at small sizes. 128x64 seems appropriate

![relay-no-min-size](https://user-images.githubusercontent.com/6320992/32137103-92d8dd84-bbe7-11e7-80e0-b182829675e4.PNG)